api/async/publish.ts

```diff
@@ -209,26 +201,16 @@ async function publishGithub(
     await uploadVersionMetaJson(
       moduleName,
       version,
-      "meta.json",
       {
         uploaded_at: new Date().toISOString(),
-        directory_listing: directory.sort((a, b) =>
-          a.path.localeCompare(b.path, "en-US")
-        ),
```

Comment:
Pretty sure this is still used by people. I would like to continue uploading this file.

Reply:
apiland should be the source of truth for that, as it does have a directory listing capability (although not flattened).
dont see the need to keep duplicate similar things around, especially when we dont need this one at all anymore

Reply:
This data is always available without requiring any database to be online. This is a good trait to have - there is no cost for us to keep this static information in S3.

Reply:
> Also, apiland gets it's directory listing from this file :)

you are right... aight will revert that change